src/plugin.ts

```diff
@@ -231,8 +231,14 @@ export class Plugin implements IPlugin {
     .map(file => {
       const p = path.parse(file)
       const topics = p.dir.split('/')
-      const command = p.name !== 'index' && p.name
+      const command = (p.name !== 'index' && p.name) ||
+        (this.type === 'core' && p.dir.length === 0 && p.name === 'index' && '')
```

Comment:
Can we assign this to a variable like `isRootIndex`, along with something like `const ROOT_INDEX_ID = ''` with your comment on top. There's a bit of magic in determining and returning this. It might be even be good to get this typed somewhere so that it can be exported as the ID to avoid loose empty strings.